cmd/rookd/api.go

```diff
@@ -31,26 +31,30 @@ import (
 )
 
 var apiCmd = &cobra.Command{
-	Use:    "api",
-	Short:  "Runs the Rook REST API service",
-	Hidden: true,
+	Use:   "api",
+	Short: "Runs the Rook API service",
 }
 var apiPort int
+var repoPrefix string
 
 func init() {
-	apiCmd.Flags().IntVar(&apiPort, "api-port", 0, "port on which the api is listening")
+	apiCmd.Flags().IntVar(&apiPort, "port", 0, "port on which the api is listening")
+	apiCmd.Flags().StringVar(&repoPrefix, "repo-prefix", "quay.io/rook", "the repo from which to pull images")
```

Comment:
Instead of repoPrefix, i would separate this and call the first part `registry` or `registry-location`. Examples of registry location is quay.io, dockerhub or 127.0.0.1:5000 if using a private registry. The `rook/rookd` should then become `repository-name` or `repository`. This is just to add more correctness to the image names.